go.mod

```diff
@@ -290,5 +290,6 @@ require (
 
 replace (
 	github.com/opencontainers/umoci => github.com/project-stacker/umoci v0.0.0-20240906174318-e9397ba4ced0
+	machinerun.io/atomfs => github.com/rchincha/atomfs v0.0.0-20250217174059-a0b78e163e7f
```

Comment:
whats in here that's not in tip atomfs?

Reply:
https://github.com/project-machine/atomfs/pull/25

make filesystems more pluggable, and all of erofs

Reply:
None of the unresolved comments have been addressed in https://github.com/project-machine/atomfs/pull/25